decidim-admin/app/assets/stylesheets/decidim/admin/modules/_table-list.scss

```diff
@@ -52,6 +52,13 @@
     }
   }
 
+  &.table-list--selectionable{
```

Comment:
Maybe this should be `selectable` instead?

Reply:
😮 😳 🙈 My spanglish played me a dirty trick